lib/ae_mdw/transfers.ex

```diff
@@ -54,6 +56,18 @@ defmodule AeMdw.Transfers do
     end
   end
 
+  defp inside_txi_range?(
+         {:txi, %Range{first: first_txi, last: last_txi}},
+         {_gen_txi, _kind, _account_pk, ref_txi}
+       )
+       when first_txi < last_txi,
+       do: not is_nil(ref_txi) and first_txi <= ref_txi and ref_txi <= last_txi
+
+  defp inside_txi_range?({:txi, %Range{last: last_txi}}, {_gen_txi, _kind, _account_pk, ref_txi}),
+    do: not is_nil(ref_txi) and last_txi <= ref_txi and ref_txi >= last_txi
```

Comment:
`last_txi <= ref_txi and ref_txi >= last_txi -> last_txi == ref_txi`

could we have a test case for this situation too?

Reply:
nice catch! I've fixed the code and added a test case for it